### PR TITLE
Tech Talk Groups

### PR DESCRIPTION
**Pull request by SheaBelsky**
## Description

This pull request allows Tech Talks on Awesome Learning to be put into groups, allowing for greater discoverability. This is separate from tags! Tags are at the Talk level, but groups contain lots of tech talks.

### CODE REVIEW

File: src/components/TechTalkGroup/TechTalkGroup.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      <List component="ol" disablePadding aria-label="Available courses">
+        {props.techTalks.map(({frontmatter: techTalk}, index) => (
+          <ListItem
+            disableGutters
+            key={techTalk.title}
+            className={classes.sectionWrapper}
+          >
+            <ListItemText
+              primary={
+                <Fragment>
```
Comment (alexcaulfield) [inline src/components/TechTalkGroup/TechTalkGroup.js:56]: super picky but you can just use `<>` to remove the import too

**Review by alexcaulfield — APPROVED** [review]

File: src/components/TechTalkGroup/TechTalkGroup.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        ))}
+      </List>
+    </Box>
+  );
+};
+
+export const techTalkGroupPropType = PropTypes.shape({
+  description: PropTypes.string,
+  slug: PropTypes.string,
+  title: PropTypes.string,
```
Comment (yleviel) [inline src/components/TechTalkGroup/TechTalkGroup.js:87]: shouldn't all of these3 of `.isRequired` or `defaultProps`?

File: src/components/TechTalkGroups/TechTalkGroups.js [hunk 3]
```diff
@@ -43,37 +43,37 @@ const TechTalks = ({techTalks}) => {
   return (
     <Box m="auto" maxWidth={theme.breakpoints.values.lg}>
       <Typography variant="h1" color="textPrimary">
-        Tech Talks
+        Tech Talk Groups
       </Typography>
-      {techTalks
+      {techTalkGroups
         .slice((page - 1) * ITEMS_PER_PAGE, page * ITEMS_PER_PAGE)
```
Comment (yleviel) [inline src/components/TechTalkGroups/TechTalkGroups.js:49]: Should this be at done at the service layer level?

File: src/components/TechTalkGroups/TechTalkGroups.js [hunk 4]
```diff
@@ -43,37 +43,37 @@ const TechTalks = ({techTalks}) => {
...
-        .map((techTalk) => (
+        .map((techTalkGroup) => (
           <Box
             display="flex"
             flexDirection="column"
-            key={techTalk.frontmatter.title}
+            key={techTalkGroup.frontmatter.title}
```
Comment (yleviel) [inline src/components/TechTalkGroups/TechTalkGroups.js:54]: do groups have ids?  if so, those could be used instead as title's could be non-unique.  Maybe `slug` is more appropriate here?

File: src/components/TechTalkGroup/TechTalkGroup.js (hunk 1 above)
Comment (SheaBelsky) [reply src/components/TechTalkGroup/TechTalkGroup.js:56]: I personally use Fragment for readability!

File: src/components/TechTalkGroup/TechTalkGroup.js (hunk 2 above)
Comment (SheaBelsky) [reply src/components/TechTalkGroup/TechTalkGroup.js:87]: The rest of the platform (everywhere) doesn't have `.isRequired`; I need to go back through and add `.isRequired` in places, but I left it out here to keep things consistent for now.

File: src/components/TechTalkGroups/TechTalkGroups.js (hunk 3 above)
Comment (SheaBelsky) [reply src/components/TechTalkGroups/TechTalkGroups.js:49]: Do you mean in this component's parent?

File: src/components/TechTalkGroups/TechTalkGroups.js (hunk 4 above)
Comment (SheaBelsky) [reply src/components/TechTalkGroups/TechTalkGroups.js:54]: Updated!

**Pull request merged**